packages/dids/tests/methods/did-jwk.spec.ts

```diff
@@ -1,15 +1,17 @@
+import type { DidDocument } from '../../src/types/did-core.js';
 import type { Jwk } from '@web5/crypto';
+import type { PortableDid } from '../../src/types/portable-did.js';
 import type { UnwrapPromise } from '@web5/common';
 
-import { expect } from 'chai';
-import { LocalKeyManager } from '@web5/crypto';
-
-import type { DidDocument } from '../../src/types/did-core.js';
-import type { PortableDid } from '../../src/types/portable-did.js';
+import chaiAsPromised from 'chai-as-promised';
+import DidJwkResolveTestVector from '../../../../web5-spec/test-vectors/did_jwk/resolve.json' assert { type: 'json' };
 
 import { DidErrorCode } from '../../src/did-error.js';
 import { DidJwk } from '../../src/methods/did-jwk.js';
-import DidJwkResolveTestVector from '../../../../web5-spec/test-vectors/did_jwk/resolve.json' assert { type: 'json' };
+import { LocalKeyManager } from '@web5/crypto';
+import { expect, use } from 'chai';
+
+use(chaiAsPromised);
```

Comment:
While communicated poorly perhaps the plan is to remove chaiAsPromised from all of the @web5 packages to clear the way for being able to migrate off Mocha and Chai at some time.

```suggestion
```